### PR TITLE
xenu: Add version 1.3.8

### PR DESCRIPTION
**Pull request by jfastnacht**

### CODE REVIEW

File: bucket/xenu.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Setup.exe\", \"$dir\\`uninst.exe\" -Force -Recurse"
+        ]
+    },
+    "shortcuts": [
+        [
+            "Xenu.exe",
+            "Xenu"
+        ]
+    ]
+}
```
Comment (pratikpc) [inline bucket/xenu.json:20]: Would need autoupdate or checkver

Comment (issaclin32) [reply bucket/xenu.json:20]: I don't think *checkver* is needed in this case.
`(current version: 1.3.8 from September 4th, 2010)`
http://home.snafu.de/tilman/xenulink.html

File: bucket/xenu.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "homepage":  "http://home.snafu.de/tilman/xenulink.html",
+    "description": "Xenu's Link Sleuth checks Web sites for broken links",
+    "version":  "1.3.8",
+    "license":  "Unknown",
+    "url": "http://home.snafu.de/tilman/XENU.ZIP#/dl.7z",
```
Comment (rashil2000) [inline bucket/xenu.json:6]: Is the dl.7z part required? It's already zip

File: bucket/xenu.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "homepage":  "http://home.snafu.de/tilman/xenulink.html",
+    "description": "Xenu's Link Sleuth checks Web sites for broken links",
+    "version":  "1.3.8",
+    "license":  "Unknown",
+    "url": "http://home.snafu.de/tilman/XENU.ZIP",
+    "hash": "sha256:319108b72709e32c250bb49bd02739f26d37b8a9d5a9bb3bbfa488d7354e197f",
```
Comment (rashil2000) [inline bucket/xenu.json:7]: `sha256:` can also be omitted

**Pull request merged**